web/package.json

```diff
@@ -27,15 +27,15 @@
     "leaflet.vectorgrid": "1.3.0",
     "browserify": "16.2.0",
     "browserify-swap": "0.2.2",
-    "d3": "5.9.7",
+    "d3": "5.11.0",
     "jquery": "3.4.1",
-    "leaflet": "1.3.1",
+    "leaflet": "1.5.1",
```

Comment:
I think also the css and index.html needs an update if leaflet js is updated

Reply:
Ah good point, there was indeed a minor css change, I updated this PR.